app/Main.hs

```diff
@@ -58,6 +58,12 @@ optionParser = Opts
   ( long "version"
   <> short 'v'
   <> help "Show version of the program" )
+  <*> strOption
+  ( long "config"
+  <> short 'c'
+  <> value "hap.yaml"
+  <> metavar "PATH"
```

Comment:
@mrkkrp could you add also [showDefault](https://github.com/pcapriotti/optparse-applicative#quick-start) please. In order to print the default value as part of help text.